lua/nvim-navbuddy/init.lua

```diff
@@ -119,7 +105,11 @@ local config = {
 		follow_node = true,
 		highlight = true,
 		reorient = "smart",
-		scrolloff = nil
+		scrolloff = nil,
+	},
+	folding = {
+		foldmethod_auto_set_manual = true,
+		leading_spaces = 2, -- to avoid the monstruosity err msg "at least two spaces before inline comment"
```

Comment:
If you want to add any folding releated options, do it like the telescope action.